client/payment-methods/index.js

```diff
@@ -202,50 +165,17 @@ const PaymentMethods = () => {
 				</WcPaySurveyContextProvider>
 			) : null }
 
-			<Card
-				className={ classNames( 'payment-methods', {
-					'is-loading': status === 'pending',
-				} ) }
-			>
-				{ isUpeEnabled && (
-					<CardHeader className="payment-methods__header">
-						<h4 className="payment-methods__heading">
-							<span>
-								{ __(
-									'Payment methods',
-									'woocommerce-payments'
-								) }
-							</span>
-						</h4>
-						<PaymentMethodsDropdownMenu
-							setOpenModal={ setOpenModalIdentifier }
-						/>
-					</CardHeader>
-				) }
-
-				{ isUpeEnabled && upeType === 'legacy' && (
-					<CardHeader className="payment-methods__header">
-						<InlineNotice
-							icon
-							status="warning"
-							isDismissible={ false }
-						>
-							{ rollbackNoticeForLegacyUPE }
-						</InlineNotice>
-					</CardHeader>
-				) }
-
-				{ ! isUpeEnabled && (
-					<CardHeader className="payment-methods__header">
-						<InlineNotice
-							icon
-							status="warning"
-							isDismissible={ false }
-						>
-							{ rollbackNoticeForLegacyCard }
-						</InlineNotice>
-					</CardHeader>
-				) }
+			<Card className={ classNames( 'payment-methods' ) }>
```

Comment:
The `classNames` function seems to have become irrelevant, here.
```suggestion
			<Card className="payment-methods">
```

Reply:
Thanks! ceeb2c98bf73206ceaea4a7ad8347d9326777437